zipkin-server/src/test/java/zipkin2/server/internal/pulsar/Access.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package zipkin2.server.internal.pulsar;
+
+import org.springframework.boot.autoconfigure.context.PropertyPlaceholderAutoConfiguration;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.Configuration;
+import zipkin2.collector.pulsar.PulsarCollector;
+
+/** opens package access for testing */
+public final class Access {
```

Comment:
This class is needed only when `ZipkinPulsarCollectorProperties` is declared as package-private (https://github.com/openzipkin/zipkin/pull/3788/files#r1929613189)